pdns/lua-base4.cc

```diff
@@ -166,7 +166,7 @@ void BaseLua4::prepareContext() {
   d_lw->registerFunction<ComboAddress(Netmask::*)()>("getNetwork", [](const Netmask& nm) { return nm.getNetwork(); } ); // const reference makes this necessary
   d_lw->registerFunction<ComboAddress(Netmask::*)()>("getMaskedNetwork", [](const Netmask& nm) { return nm.getMaskedNetwork(); } );
   d_lw->registerFunction("isIpv4", &Netmask::isIpv4);
-  d_lw->registerFunction("isIpv6", &Netmask::isIpv6);
```

Comment:
The existing version is documented in docs/lua-records/reference/netmask.rst and pdns/recursordist/docs/lua-scripting/netmask.rst so we probably should not remove it right away. We could perhaps add the new version and deprecate the existing one?